cli.js

```diff
@@ -96,5 +97,18 @@ function handleFkillError(processes) {
 if (cli.input.length === 0) {
 	init(cli.flags);
 } else {
-	fkill(cli.input, cli.flags).catch(() => handleFkillError(cli.input));
+	let pids = [];
+	const promises = [];
+	for (let i = 0; i < cli.input.length; i++) {
+		if (cli.input[i][0] === ':') {
+			promises.push(portToProc(parseInt(cli.input[i].slice(1), 0)));
+		} else {
+			pids.push(cli.input[i]);
+		}
+	}
+	Promise.all(promises).then(promisedPids => {
+		promisedPids = promisedPids.filter(pid => pid);
```

Comment:
Yeah, maybe. I usually throw but I see where you coming from. Maybe @sindresorhus @SamVerschueren has any thoughts?